EIPS/pay_req_url_fmt.md

```diff
@@ -0,0 +1,65 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: URL Format for Transaction Requests
+    Author: Daniel A. Nagy <daniel@ethereum.org>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Replaces: 67
+    Created: 2017-08-01
+    Requires: 20
+
+## Simple Summary
+A standard way of representing various transactions, especially payment requests in Ethers and ERC #20 tokens as URLs.
+
+## Abstract
+URLs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications. A standardized URL format for payment requests allows for instant invocation of the user's preferred wallet application (even if it is a webapp or a swarm đapp), with the correct parameterization of the payment transaction only to be confirmed by the (authenticated) user.
+
+## Motivation
+The convenience of representing payment requests by standard URLs has been a major factor in the wide adoption of Bitcoin. Bringing a similarly convenient mechanism to Ethereum would speed up its acceptance as a payment platform among end-users. In particular, URLs embedded in broadcast Intents are the preferred way of launching applications on the Android operating system and work across practically all applications. Desktop web browsers have a standardized way of defining protocol handlers for URLs with specific protocol specifications. Other desktop applications typically launch the web browser upon encountering a URL. Thus, payment request URLs could be delivered through a very broad, ever growing selection of channels.
+
+This specification supersedes ERC #67, which is a URL format for representing arbitrary transactions in a low-level fashion. This ERC focuses specifically on the important special case of payment requests, while allowing for other, ABI-specified transactions.
+
+## Specification
+
+### Syntax
+Payment request URLs contain "ethpay" in their schema (protocol) part and are constructed as follows:
+
+    request                 = "ethereum" ":" target_address [ "/" function_name ] [ "?" parameters ]
```

Comment:
What happens if you want to mix function parameters and non function ones? Say I want an url to a crowd sale and I want to request payment? My suggestion would be to always add the non function parameters before the /function_name:

```
ethereum:hall-of-fame-ico.eth?value=1e18/hallOfFame?string="bob"
```

Reply:
I don't like that. I'd rather add the non-function parameters at the end, but even that would be only a convention, not a requirement. However, the function name should always come first, I think. Otherwise, it would become too easy to write ambiguous-looking URLs where it is hidden between similarly looking strings. Also, it would make it less URL-like.

Reply:
But if you do that, you'd need some way to differentiate it from a function parameter called "value". One other option would have to call it also a less generic name like "txValue": before the /function_name either can be used, but after /function_name only txValue is accepted